src/modules/comments.js

```diff
@@ -0,0 +1,44 @@
+import commentsCounter from './commentsCounter.js';
+
+const showPopups = async (data, updateComment) => {
+  const comments = await updateComment(data.name);
+
+  const commentNum = commentsCounter(comments);
+
+  const content = document.querySelector('.popup-section');
+
+  content.innerHTML = `
+<div class ="movie-detail">
+  <button type="button" class="close">
+    <i class="fa fa-times fa-1x" id="close"></i>
+  </button>
+  <div class="movie-image">
+      <img src=${data.image.medium}  />
+  </div>
+  <div class="comment-description">
+      <h2>${data.name}</h2>
+      <p>${data.summary}</p>
+  </div>
+  <div class="show-comment">
+    <h3> Comments (${commentNum || 0})</h3>
+  <ul class="comments-list">${comments
+    .map(
```

Comment:
I suggest you destructure the data object as it makes your code cleaner

Reply:
Thanks for your review 👍 

I have no idea on how to go about that